ui/src/admin/components/chronograf/NewUserTableRow.js

```diff
@@ -0,0 +1,217 @@
+      provider: userProvider,
+      scheme: userScheme,
+      superAdmin: userSuperAdmin.value,
+      roles: [
+        {
+          name: userRole,
+          organization: userOrganization.id,
+        },
+      ],
+    }
+
+    onCreateUser(newUser)
+    onCancelCreateUser()
+  }
+
+  handleInputFocus = e => {
+    e.target.select()
+  }
+
+  handleSelectRole = newRole => {
+    this.setState({userRole: newRole.text})
+  }
+
+  handleSelectSuperAdmin = userSuperAdmin => {
+    this.setState({userSuperAdmin})
+  }
+
+  handleSelectOrganization = newUserOrganization => {
+    this.setState({userOrganization: newUserOrganization})
+  }
+
+  render() {
+    const {
+      colOrg,
+      colRole,
+      colProvider,
+      colScheme,
+      colSuperAdmin,
+      colActions,
+    } = USERS_TABLE
+    const {
+      onCancelCreateUser,
+      userRoles,
+      organizations,
+      currentOrganization,
+    } = this.props
+    const {
+      userName,
+      userProvider,
+      userScheme,
+      userRole,
+      userSuperAdmin,
+      userOrganization,
+    } = this.state
+
+    const allowCreate =
+      !userName || !userProvider || !userRole || !userOrganization
+
+    const isDefaultOrg = currentOrganization.id === DEFAULT_ORG_ID
+
+    const organizationsMinusDefault = organizations.filter(org => {
+      if (org.id !== DEFAULT_ORG_ID) {
+        return org
+      }
+    })
+    return (
+      <tr className="chronograf-admin-table--new-user">
+        <td className="chronograf-admin-table--check-col" />
+        <td>
+          <input
+            className="form-control input-xs"
+            type="text"
+            placeholder="OAuth Username..."
+            autoFocus={true}
+            value={userName}
+            onChange={this.handleInputChange('userName')}
+          />
+        </td>
+        <td style={{width: colOrg}}>
+          {isDefaultOrg
+            ? <Dropdown
+                items={organizationsMinusDefault.map(org => ({
+                  ...org,
+                  text: org.name,
+                }))}
+                selected={
+                  userOrganization.id === DEFAULT_ORG_ID
+                    ? 'Choose one'
+                    : userOrganization.name
+                }
+                onChoose={this.handleSelectOrganization}
+                buttonColor="btn-primary"
+                buttonSize="btn-xs"
+                className="dropdown-stretch"
+              />
+            : <span className="chronograf-user--org">
+                {currentOrganization.name}
+              </span>}
+        </td>
+        <td style={{width: colRole}}>
+          <Dropdown
+            items={userRoles.map(role => ({...role, text: role.name}))}
+            selected={userRole || 'Assign'}
+            onChoose={this.handleSelectRole}
+            buttonColor="btn-primary"
+            buttonSize="btn-xs"
+            className="dropdown-80"
+            disabled={userOrganization.id === DEFAULT_ORG_ID}
```

Comment:
how about a function called `isDefaultOrganization` that you just pass any organization ID into and can be used throughout your code, since this function is done a number of times?